wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Linker.kt

```diff
@@ -348,17 +348,17 @@ class Linker {
     requestedFields.add(field)
   }
 
-  /** Returns the field named [field] on the message type of [self]. */
+  /** Returns the field named [field] on the message type of [protoType]. */
   fun dereference(
-    self: Field,
```

Comment:
I "inlined" its usage because map fields don't have corresponding `Field`s but only rely on the `ProtoType` of their key and value.